text/0119-instance-options.md

```diff
@@ -449,6 +450,22 @@ Activates SSL session resumption.
 
 Activates SSL tickets.
 
+
+#### 3.3.29. Max results range
+
+**Environment variable**: `MEILI_MAX_RESULTS_RANGE`
+**CLI option**: `--max-results-range`
+**Default**: `1000`
```

Comment:
Plus, it means you cannot set it by index

Reply:
@curquiza, yes in the end this is a value that we don't really wantt the user to change, we set it to a default value, and leave it like that. If we find that people complain and REALLY want to set it by index, we'll iterate

Reply:
To be honest, I was surprised to find this setting as an instance option. From my POV, it does not look to be from the same scope as the other options, like schedule snapshots, set up the DB size, the path of the DB, the master key etc... but more a setting for the search like `searchableAttributes`, `synonyms`...etc

Reply:
And speaking about a first iteration, I would say, let's not define any settings, and if users really want to change it, we will add it later. But maybe too "risky" for the user experience

Reply:
Based on what @MarinPostma said, and the last @curquiza message, I agree that it should not be an option since the requirement is a **security** concern. If we keep the option, we **must warn** the users about what they are doing (like most of the other configurations 😄).

In any case, if I understood correctly (please tell me if I'm wrong)
On the Elasticsearch side, they had [`index.max_result_window`](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-max-result-window) to configure the same behavior.

Giving more light to the situation, we could have the hardcoded limit AND provide a way for users to adopt a new value if they want since the value is lower than the hardcoded limit.

Reply:
That's the idea, provide a default value, and make it as hard as possible to change so we are sure the user know what they are doing. We could just have a hard codes value, but we'd get complaints, so the cli option seems like a reasonable option for a first iteration. 

That being said, I'm completely open to make it an index setting if this is desired, and we have a user need for it, but the main concern now is security